### ISSUE TITLE
Mention Channels

### ISSUE THREAD

**Issue by Deniz582** [Vencord/plugin-requests#11]
### Plugin Description

adds a button that puts the mention for the channel clicked in your message, like Discord does for users.

### Existing Plugin for other mod

https://github.com/Setloth/MentionChannels/blob/main/MentionChannels.plugin.js

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by cheesesamwich** [comment]
making this rn

**Comment by cheesesamwich** [comment]
![Discord_FiIQsuIN17](https://github.com/Vencord/plugin-requests/assets/149597648/04a7682a-94e9-4aaf-b0d5-d85ff9b1f05c)
do you mean like this?

**Comment by Deniz582** [comment]
> ![Discord_FiIQsuIN17](https://private-user-images.githubusercontent.com/149597648/304491395-04a7682a-94e9-4aaf-b0d5-d85ff9b1f05c.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4NDQwNTksIm5iZiI6MTcwNzg0Mzc1OSwicGF0aCI6Ii8xNDk1OTc2NDgvMzA0NDkxMzk1LTA0YTc2ODJhLTk0ZTktNGFhZi1iMGQ1LWQ4NWZmOWIxZjA1Yy5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxM1QxNzAyMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZmM0MGM3MDk0OTM5ZmQ3ZWZjNjA1OTNhNzdmMWU1NDkyZjljOTJiOGRkMGQ2MTQ1NjI3YmRjYWQ5MDJiOGNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3NaLrxnOWgcYCXLD3vtGewj84X-LkX8t0Q_KykT5D7I) [ ![Discord_FiIQsuIN17](https://private-user-images.githubusercontent.com/149597648/304491395-04a7682a-94e9-4aaf-b0d5-d85ff9b1f05c.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4NDQwNTksIm5iZiI6MTcwNzg0Mzc1OSwicGF0aCI6Ii8xNDk1OTc2NDgvMzA0NDkxMzk1LTA0YTc2ODJhLTk0ZTktNGFhZi1iMGQ1LWQ4NWZmOWIxZjA1Yy5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxM1QxNzAyMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZmM0MGM3MDk0OTM5ZmQ3ZWZjNjA1OTNhNzdmMWU1NDkyZjljOTJiOGRkMGQ2MTQ1NjI3YmRjYWQ5MDJiOGNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3NaLrxnOWgcYCXLD3vtGewj84X-LkX8t0Q_KykT5D7I) ](https://private-user-images.githubusercontent.com/149597648/304491395-04a7682a-94e9-4aaf-b0d5-d85ff9b1f05c.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4NDQwNTksIm5iZiI6MTcwNzg0Mzc1OSwicGF0aCI6Ii8xNDk1OTc2NDgvMzA0NDkxMzk1LTA0YTc2ODJhLTk0ZTktNGFhZi1iMGQ1LWQ4NWZmOWIxZjA1Yy5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxM1QxNzAyMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZmM0MGM3MDk0OTM5ZmQ3ZWZjNjA1OTNhNzdmMWU1NDkyZjljOTJiOGRkMGQ2MTQ1NjI3YmRjYWQ5MDJiOGNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3NaLrxnOWgcYCXLD3vtGewj84X-LkX8t0Q_KykT5D7I) [ ](https://private-user-images.githubusercontent.com/149597648/304491395-04a7682a-94e9-4aaf-b0d5-d85ff9b1f05c.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc4NDQwNTksIm5iZiI6MTcwNzg0Mzc1OSwicGF0aCI6Ii8xNDk1OTc2NDgvMzA0NDkxMzk1LTA0YTc2ODJhLTk0ZTktNGFhZi1iMGQ1LWQ4NWZmOWIxZjA1Yy5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxM1QxNzAyMzlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZmM0MGM3MDk0OTM5ZmQ3ZWZjNjA1OTNhNzdmMWU1NDkyZjljOTJiOGRkMGQ2MTQ1NjI3YmRjYWQ5MDJiOGNhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3NaLrxnOWgcYCXLD3vtGewj84X-LkX8t0Q_KykT5D7I) do you mean like this?

yes most defo and thank you so much this is very useful for me as i guide members to channels

**Comment by cheesesamwich** [comment]
alright cool, ill pull request it now (no guarantee it gets added though, its kind of a niche use case.)

**Comment by Deniz582** [comment]
totally respect the fact that you at least made one - i hope it gets added and thank you very much.
